spec/integration/collection_association_spec.rb

```diff
@@ -42,6 +42,22 @@ class Book < ActiveFedora::Base
     end
   end
 
+  describe "#delete" do
+    context "when given items not in collection" do
+      it "should return an empty set" do
+        expect(library.books.delete(Book.new)).to eq []
+      end
+      it "should not act on it" do
+        b = Book.new
+        allow(b).to receive(:persisted?).and_return(true)
+
+        library.books.delete(b)
+
+        expect(b).not_to have_received(:persisted?)
```

Comment:
> I guess it doesn't matter, just what I'm used to.

Do we want to make a style decision and pick a preference moving forward? We prefer spies because it separates out setup and assertions.